src/LiveQuery/ParseLiveQueryServer.js

```diff
@@ -170,8 +170,11 @@ class ParseLiveQueryServer {
             };
             const trigger = getTrigger(className, 'afterEvent', Parse.applicationId);
             if (trigger) {
-              const auth = await this.getAuthForSessionToken(res.sessionToken);
-              res.user = auth.user;
+              const sessionToken = this.getSessionFromClient(client, requestId);
```

Comment:
Theses 4 added code bits seems to be (almost) the same. Could you refactor this to reduce duplicate code?